po/common/common.pot

```diff
@@ -7491,3 +7491,10 @@ msgctxt "action_report_image_to_nutripatrol"
 msgid "Report this image to our moderators"
 msgstr "Report this image to our moderators"
 
+msgctxt "bottom_content_split"
+msgid "<<site_name>> is made by a non-profit association, independent from the industry. It is made for all, by all, and it is funded by all. You can support our work by <a href=\"https://world.openfoodfacts.org/donate-to-open-food-facts?utm_source=login-open-food-facts\">donating to Open Food Facts</a>.<br/><b>Thank you!</b>"
```

Comment:
I think we should remove the links in the text, and put the link only in the "donate" image above. That will simplify a lot the string to translate.